### PR TITLE
960-endoscopy-inbody-classification-tutorial

### LINKED ISSUE #960: Add tutorial notebook for endoscopy inbody classification
**Is your feature request related to a problem? Please describe.**
Would like to add a tutorial notebook for endoscopy inbody classification task.

### PR DESCRIPTION
**Pull request by binliunls**
Signed-off-by: binliu <binliu@nvidia.com>

Fixes #960  .

### Description
Add a tutorial of endoscopy inbody classification, in order to detect whether the endoscopy camera is inside patients' body. This may help to protect privacy of patients during surgical operation.  

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by binliunls** [comment]
Hi @Nic-Ma @wyli :
     Could you please help me review this endoscopy inbody classification tutorial.
Thanks,
BIn

**Review by Nic-Ma — CHANGES_REQUESTED** [review]
Pending on the legal verification.

Thanks.

**Comment by jin-nvidia** [comment]
Suggestion: After the printout `print(f"There are total {len(sample_list)} samples")`  to also add printouts of how many are assigned to the training set and how many are assigned to the validation set

**Comment by Nic-Ma** [comment]
Could you please help update the `endoscopy` to `endoscopic` in the notebook tiple and README?

Thanks in adavnce.

**Review by Nic-Ma — APPROVED** [review]

**Review by Nic-Ma — APPROVED** [review]

**Review by wyli — CHANGES_REQUESTED** [review]
I'll move the testing data to github storage

**Review by wyli — APPROVED** [review]
